src/Core/Components/DataGrid/FluentDataGrid.razor.cs

```diff
@@ -765,23 +765,23 @@ public async Task OnKeyDownAsync(FluentKeyCodeEventArgs args)
         //return Task.CompletedTask;
     }
 
-    internal async Task SetColumnWidthDiscreteAsync(int? columnIndex, float widthChange)
+    public async Task SetColumnWidthDiscreteAsync(int? columnIndex, float widthChange)
```

Comment:
Why do you want to move these methods to `public` ?
If this is the case, we will need to support these features and you need to add comments.

Reply:
For Aspire, it is needed to call these methods from the outside. I'll document them.

Reply:
Done